benchmark/daisy/daisy.jl

```diff
@@ -23,20 +23,20 @@ for name in keys(DAISY_1D)
     RELPREC[name] = Dict()
     dom = DAISY_1D[name]["dom"]
     ref = DAISY_1D[name]["ref"]
-    for m in [2, 5, 10]
-        x = Taylor1(m)
+    for ord in [2, 5, 10]
+        x = TaylorModel1(Taylor1(2*ord), 0..0, Interval(mid(dom)), dom)
```

Comment:
This line should read `x = TaylorModel1(Taylor1(ord), 0..0, Interval(mid(dom)), dom)`; no need to `2*ord` here.

Reply:
thanks for pointing that out, I will try will "ord" and check ones if we are good with relative precision.

Reply:
My guess is that the precision will be worst, since you'll be using a "lower" order than before.